swingset-demo/src/main/java/com/nqadmin/swingset/demo/MainClass.java

```diff
@@ -635,11 +636,15 @@ private static void usage() {
 	 * @param _args - optional command line arguments, which are ignored by this
 	 *              program
 	 */
+	@SuppressWarnings("UseOfSystemOutOrSystemErr")
 	public static void main(final String[] _args) {
```

Comment:
I was going to ask about logger vs system.x.print, but in a main method I think sending feedback to the terminal makes sense.

Reply:
Note that outputting the JDK version info has been there for year(s).